src/frontends/pytorch/src/op/convnd.cpp

```diff
@@ -51,6 +51,9 @@ OutputVector translate_convnd(const NodeContext& context) {
     }
     if (!context.input_is_none(2)) {
         auto bias = context.get_input(2);
+        if (std::dynamic_pointer_cast<v0::Constant>(context.get_input_from_visible_context(2).get_node_shared_ptr())) {
```

Comment:
Would be better to call `get_input_from_visible_context` once also please add at least one test

Reply:
Done